src/tools/pontoon/translate.md

```diff
@@ -62,89 +49,64 @@ There are two types of quality check failures: errors and warnings.
-
-![Interactive Hamburger menu](../../assets/images/pontoon/translate/interactive_features.png)
+Certain types of checks reported by the Translate Toolkit library may result in many false positives. For this reason, they can be [completely disabled](users.md#editor-settings).
 
 ## Performing batch actions on strings
 
-Translators can perform mass actions on batches of strings by clicking on the square to the left of any string in the sidebar. When clicked, the square will show a checkmark and the editing space will show the mass action window.
+Mass actions can be performed on multiple strings at once by clicking on the square to the left of any string in the sidebar. When clicked, the square will show a checkmark and the editing space will show the mass action panel. Note that mass actions are only available to users with [translator permissions](users.md#user-roles).
 
 ![Mass action window](../../assets/images/pontoon/translate/batch_actions.png)
 
-After selecting a string, it’s possible to hold `SHIFT` and click on another string to select a range of strings.
-
-At the top-left of the mass action window, the translator can return to the editing space by clicking on the `x` icon. To the right of the exit icon, the number of strings currently selected is displayed. At the top-right of the mass action window, the translator can select all of the strings in the sidebar.
+To select a range of strings, select the first one and hold `SHIFT`, then select the last one. To select all strings, click `SELECT ALL` on the top-right of the mass action panel. The number of selected strings will appear on the top-left, next to the Exit icon.
 
-In the `REVIEW TRANSLATIONS` section, the translator can approve suggestions or reject suggestions for all selected strings. Upon clicking `APPROVE ALL` or `REMOVE ALL SUGGESTIONS`, the label of the button will be replaced with the number of affected translations.
+To return back to the default editing space, click the Exit icon on the top-right of the mass action panel.
 
-In the `REPLACE IN TRANSLATIONS` section, the translator can input a series of characters to search for, and a series of characters to use as a replacement. This is a basic find and replace feature that will work only on the selected strings.
+In the `REVIEW TRANSLATIONS` section, the user can approve or reject suggestions for all selected strings. Upon clicking `APPROVE ALL` or `REJECT ALL SUGGESTIONS`, the label of the button will be replaced with the number of affected translations.
+Note that `APPROVE ALL` accepts the latest suggestion, but doesn’t reject other suggestions if available.
 
-Note that mass actions are only available to users with [translators rights](users.md#user-roles).
+In the `FIND & REPLACE IN TRANSLATIONS` section, the user can input the text to search for, and the text to replace it with. This is a basic find and replace feature that will work only on the selected strings.
```

Comment:
Nit: Do we need _the_ in "input the text" and "and the text"?

Reply:
I think so. The alternative would be to use `and its replacement`, but it's less clear than the current text.